src/index.js

```diff
@@ -1,14 +1,17 @@
 const express = require('express');
 
-const { contributors, faq } = require('./routes');
-const { server: { port, host } } = require('./config');
-
 // ENV configuration
 require('dotenv').config();
 
+const { contributors, faq } = require('./routes');
+const { server: { port, host } } = require('./config');
+
 // Database connection
 require('./helpers/databaseConnection');
 
+// Seed DB
+require('./tasks/seedFaq');
```

Comment:
No need to require this here. Lets add a script to the `package.json` called `seed` that can be run as needed.

Reply:
Got it now, Done.

Reply:
See latest commit, is that the flow you had in mind?